apps/redi-connect/src/pages/app/applications/MobileView.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import Select from 'react-select'
+import { useSelector } from 'react-redux'
+
+import { RootState } from '../../../redux/types'
+import { Button } from '@talent-connect/shared-atomic-design-components'
+import { customStyles } from '../../../../../../libs/shared-atomic-design-components/src/lib/atoms/SelectStyles'
+import ApplicationCard from '../../../components/organisms/ApplicationCard'
+import './MobileView.scss'
+
+const applicationStatuses = [
+  { value: 'pending', label: 'Pending' },
+  { value: 'accepted', label: 'Accepted' },
+  { value: 'declined', label: 'Declined' },
+  { value: 'cancelled', label: 'Cancelled' },
+]
+
+const MobileView = ({
```

Comment:
@katamatata, I think we mostly declare components in this codebase using `function ComponentName` over `const ComponentName = `. Please verify and let's maximize consistency in code style.

Reply:
I would say it's the other way around:) Almost all the components have arrow function syntax. For example, all components in the _molecules_ and _organisms_ folders. I think only a few older ones (including the Applications component) were declared using  `function ComponentName` syntax. So, we probably would need to change the older ones. What do you think?

Reply:
Ah, a nice mix then. Then just leave it for now, and we can create a refactoring ticket for this.